python/tvm/target/target.py

```diff
@@ -188,7 +188,8 @@ def arm_cpu(model='unknown', options=None):
         "p20":       ["-model=kirin970", "-mtriple=arm64-linux-android", "-mattr=+neon"],
         "p20pro":    ["-model=kirin970", "-mtriple=arm64-linux-android", "-mattr=+neon"],
         "rasp3b":    ["-model=bcm2837", "-mtriple=armv7l-linux-gnueabihf", "-mattr=+neon"],
-        "rasp4b":    ["-model=bcm2711", "-mtriple=arm-linux-gnueabihf", "-mattr=+neon"],
+        "rasp4b":    ["-model=bcm2711", "-mtriple=armv8l-linux-gnueabihf", "-mattr=+neon"],
+        "rasp4b64":  ["-model=bcm2711", "-mtriple=aarch64-linux-gnu", "-mattr=+neon"],
```

Comment:
I think we should add `-mcpu=cortex-a72` for rasp4. We should get better performance after it than generic cpu instruction.

Reply:
thanks, I've added this additional field.